examples/example.hpp

```diff
@@ -1072,7 +1072,7 @@ void register_glfw_callbacks(window& app, glfw_state& app_state)
 }
 
 // Find devices with specified streams
-std::string depth_with_stream_type_present(rs2_stream type)
+void device_with_streams(rs2_stream type, std::string& out_serial)
```

Comment:
Change function's return type to `bool` for the code to be more fluent.
Make the 1st param a vector to support for multiple requests